iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Branch.java

```diff
@@ -140,16 +143,81 @@
 
     Side getSide(Terminal terminal);
 
-    CurrentLimits getCurrentLimits(Side side);
+    default List<OperationalLimits> getOperationalLimits1() {
+        return getCurrentLimits1() != null ? Collections.singletonList(getCurrentLimits1()) : Collections.emptyList();
+    }
 
     CurrentLimits getCurrentLimits1();
 
+    default ActivePowerLimits getActivePowerLimits1() {
+        return null;
+    }
+
+    default ApparentPowerLimits getApparentPowerLimits1() {
+        return null;
+    }
+
     CurrentLimitsAdder newCurrentLimits1();
 
+    default ActivePowerLimitsAdder newActivePowerLimits1() {
+        throw new UnsupportedOperationException();
```

Comment:
Still the same discussion: I suppose you will use this new method (and the others) in the converters. Suppose you use a different implementation of IIDM, this will crash the import.

I don't know if we should care about it, or if we have to change our strategy for defaulted methods, like providing a mock that does nothing but ensure that the importers work with not updated implementations

Reply:
To be honest, not sure what it is best for whole new objects... I worry that adding mock objects will only make the API more confusing

Reply:
The mock is not the good solution: what I mean is implementing the new methods with a throw will hide the issue. You don't break the compilation but it will fail at runtime that it's worst.